lib/cocoapods-playgrounds/workspace.rb

```diff
@@ -168,8 +174,14 @@ def generate_project
     def generate_swift_code(path)
       File.open(path + 'Contents.swift', 'w') do |f|
         f.write("//: Please build the scheme '#{target_name}' first\n")
-        f.write("import XCPlayground\n")
-        f.write("XCPlaygroundPage.currentPage.needsIndefiniteExecution = true\n\n")
+        if @use_legacy_xcode
```

Comment:
@janiduw It could be done this way so it doesn't depend on the Xcode version used to generate the playground:
```
#if swift(>=3.0)
    import PlaygroundSupport
    PlaygroundPage.current.needsIndefiniteExecution = true
#else
    import XCPlayground
    XCPlaygroundPage.currentPage.needsIndefiniteExecution = true
#endif
```